file_store/src/file_sink.rs

```diff
@@ -145,10 +157,12 @@ impl FileSinkBuilder {
 pub struct FileSinkClient {
     sender: MessageSender,
     metric: &'static str,
+    shutdown_listener: triggered::Listener,
 }
 
 const OK_LABEL: Label = Label::from_static_parts("status", "ok");
 const ERROR_LABEL: Label = Label::from_static_parts("status", "error");
+const SEND_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(10);
```

Comment:
10 secs seems a very long time to wait for an internal msg send

Reply:
yeah, I just picked something to get a conversation going, what do you think it should be?

Reply:
5 secs ? good old erlang default sync timeout

Reply:
Fixed bff5af5